File: ExoAPI/Entitie/Product.cs
using ExoAPI.Type;

namespace ExoAPI.Entitie;

public class Product
{
    public int Id { get; set; }
    public string Origin { get; set; }
    public string Name { get; set; }
    public int Quantite  { get; set; }
    public Usages Usage { get; set; }
}

File: testquasar/src/models/product.ts
export interface ProductDto {
  id: number;
  origin: string;
  name: string;
  quantite: number;
  usage: Usages;
}

export enum Usages {
  Autre = 0,
  Informatique = 1,
  Transport = 2,
  Recherche = 3,
}
